### DISCUSSION TITLE
Temp mail service

### DISCUSSION THREAD

**Discussion by ghost** [inbucket/inbucket#239] [Q&A] [answered]
Sorry for my really nooby question, but is it possible to setup Inbucket and use as a temporary email service ? And be able to read the emails received? Thanks by advance.

**Comment by jhillyerd** [comment, marked as answer]
Yes, you can definitely do that.  I typically recommend fronting Inbucket with a traditional mail server like Postfix rather than exposing it to the internet directly.  If you don't do that, make sure to set INBUCKET_SMTP_ACCEPTDOMAINS so it does not appear as an open relay to spammers.

Inbucket does not offer any password protection, so if you don't want other folks looking at/deleting the messages you'll want to put it behind a proxy server that offers authentication.

**Reply by ghost** [reply]
Hey thanks for the quick answer, will definitely try that out, much appreciated.
Could I use some smtp servers found on GitHub as a replacement to Postfix? 
 Have a good day :)

Edit: nvm I didn’t understood your answer well. I don’t think I will need smtp server since I only have to receive emails from one domain. 
This were I struggled, I might be dumb but I didn’t understood how to setup Inbucket as a receiving server.

**Reply by jhillyerd** [reply]
I think you could use any mail server that supports a [smart relay host](https://en.wikipedia.org/wiki/Smart_host), and then set your Inbucket address+port as that host.

**Reply by ghost** [reply]
Will look into thank you so much !
